DEPS

```diff
@@ -197,18 +197,3 @@ include_rules = [
   "-brave/third_party/bitcoin-core",
   "-brave/third_party/argon2",
 ]
-
-# Temporary workaround for massive nummber of incorrect test includes
-specific_include_rules = {
```

Comment:
this was allowing new browser tests in components and also layering violations in unit tests